src/console_cmds.cpp

```diff
@@ -2225,11 +2225,11 @@ static void OutputContentState(const ContentInfo &ci)
 
 static bool ConContent(std::span<std::string_view> argv)
 {
-	static ContentCallback *cb = nullptr;
-	if (cb == nullptr) {
-		cb = new ConsoleContentCallback();
-		_network_content_client.AddCallback(cb);
-	}
+	[[maybe_unused]] static ContentCallback *const cb = []() {
```

Comment:
```suggestion
	[[maybe_unused]] static ContentCallback * const cb = []() {
```
Prevalent coding style puts a space between the asterisk and const.